README.md

```diff
@@ -1 +1,131 @@
-# Leaderboard
\ No newline at end of file
+<a name="LeaderBoard"></a>
+<div align="center">
+  <br/>
+  <h3><b>LeaderBoard</b></h3>
+</div>
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[To_Do_List] \_To\_Do\_List]
+  - [🛠 Built With Html, css, javascript](#-built-with-html-css-javascript)
+    - [Tech Stack HTML5, CSS3 JAVASCRIPT](#tech-stack-html5-css3-javascript)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author(s) ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+  - [📝 License](#-license)
+
+# 📖 [LeaderBoard] <a name="LeaderBoard">LeaderBoard</a>
+
+A solo programming project that highlights the use of Webpack to bundle JavaScript, ES6 syntax, API, callbacks and promise
+
+## 🛠 Built With <a name="built-with">Html, css, javascript</a>
+
+### Tech Stack <a name="tech-stack">HTML5, CSS3 JAVASCRIPT</a>
+
+HTML5 and css3 which are the latest versions and JavaScript were used in this project.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+-  leaderboard built with HTML CSS and javascript 
+-  Dynamically loading with javascript.
+-  Apply module bundler (Webpack).
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+ Get your pc.
+ Clone this project to your local machine. 
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+Basic knowledge of computers. To run this project you need to run (npm run serve ).
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+### Install
+
+Install this project with cd to ToDoList.
+
+### Run tests
+
+To run tests, run the following command:
+npm run serve  
+
+
+### Deployment
+
+You can deploy this project using: GitHub pages.
+
+<!-- AUTHORS -->
+
+## 👥 Author(s) <a name="authors"></a>
+
+> Walid Kabboussa
+
+👤 **Author1**
+
+- GitHub: [@Hisoka37](https://github.com/Hisoka37)
+- Twitter: [@KBwalid9](https://twitter.com/KBwalid9)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/walidkb/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+- **[future_feature_1]**  add Score to the table.
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Hisoka37/To_Do_List).
```

Comment:
- Please, let's update the link to the issues page to be the link to this current project's issue page as the link is broken.